### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by michaelizergit** [zubairq/appshare#25]
Greetings, @zubairq 

I'm a designer in development and an open source enthusiast. I was checking out projects and noted that you may want a logo for yours. What do you think? It would be totally free and under your instructions, and you can quit whenever you want.

Best regards.

**Comment by zubairq** [comment]
Sure Michael. Can you design a logo for appshare.co?

**Comment by michaelizergit** [comment]
Yes, that's the idea. Anything you want to highlight? any preferences for a color or something special?

**Comment by zubairq** [comment]
Well, appshare is aimed at large companies over 10,000 people, so you decide

**Comment by michaelizergit** [comment]
Hello, I'm so sorry for disappearing I was finishing a semester. I want to resume the work as soon as possible just passing by to ask if you are still interested.

**Comment by zubairq** [comment]
Sure, you can still try
